CHANGES.md

```diff
@@ -25,6 +25,9 @@ To be released.
         `IImmutableDictionary<Address, IValue>` (was `AddressStateMap`).
      -  `IStore.SetBlockStates()` method became to take
         `IImmutableDictionary<Address, IValue>` instead of `AddressStateMap`.
+ -  `Swarm<T>.PreloadAsync()` method and `Swarm<T>.StartAsync()` method became
+    to receive `preloadBlockDownloadFailed` event handler as a parameter.
```

Comment:
Unsure which is good: “as an argument” or “through a parameter” though.  🤔

```suggestion
    to take `preloadBlockDownloadFailed` event handler as an argument.
```

Reply:
“as an argument” looks better to me.